### ISSUE TITLE
Fail to deploy a folder to the storage account with an error

### ISSUE THREAD

**Issue by v-chenzha** [microsoft/vscode-azurestorage#1261]
**OS:** Win10
**Build Version:** 20230801.1
**Regression:** Previous Release

**Repro Steps:**

1. Deploy a folder to storage account.
2. Check whether succeeds to deploy a folder to the storage account.

**Expect:**
Succeed to deploy a folder to the storage account.

**Actual:**
Fail to deploy a folder to the storage account with an error.
![image](https://github.com/microsoft/vscode-azurestorage/assets/107091828/40243b2e-e3bf-47ca-8d0f-08a4c1c92bfe)

Details in Output:
```
Deploying to static website "/subscriptions/3b55f2b8-b6d0-4e03-a3f9-2b22471450c6/resourceGroups/cindytest1/providers/Microsoft.Storage/storageAccounts/cindytest1/$web"
Querying Azure... Method: listBlobsFlat blobContainerName: "$web" prefix: ""
Uploading from "c:\Users\AzureTest\Desktop\New folder (2)" to "$web"
Error: Cannot find module '@azure-tools/azcopy-win32'
Require stack:
- c:\Users\AzureTest\.vscode\extensions\ms-azuretools.vscode-azurestorage-0.15.4-alpha.0\dist\extension.bundle.js
- c:\Users\AzureTest\.vscode\extensions\ms-azuretools.vscode-azurestorage-0.15.4-alpha.0\main.js
- c:\Users\AzureTest\AppData\Local\Programs\Microsoft VS Code\resources\app\out\vs\loader.js
- c:\Users\AzureTest\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-amd.js
- c:\Users\AzureTest\AppData\Local\Programs\Microsoft VS Code\resources\app\out\bootstrap-fork.js
```

**More Info:**
This issue also reproduces when uploading a file/folder to blob container.

**Comment by alexweininger** [comment]
This should be fixed by #1309, but waiting for confirmation from @v-xinda.

**Comment by v-xinda** [comment]
@alexweininger  This issue is fixed on Mac and Linux platforms, it doesn't reproduce for now.
